kolko.py

```diff
@@ -0,0 +1,66 @@
+
+def printer(tab):
+    'print table with positions to choose'    
+    print('{}\n{}\n{}'.format(tab[0],tab[1],tab[2]))
+
+def instert_to_tab(player,n):
+    '''inserting player to table return update table
+        player - X or O
+        n - position number'''    
+    c=0
+    for arr in tab:
+        for i,pos in enumerate(arr):
+            if n==c:
+                arr[i]=player
+            c += 1
+    return tab
+
+def winner(tab):    
+    '''check who win. Return True when anybody win.
+    tab - used_nums'''
+    for arr in way_to_win:
+        if players[0]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]] or players[1]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]]:
```

Comment:
Tak dlugie linie nie są czytelne. Dobrą praktyką jest nie przekraczanie 80 lub 120 znaków linii. Z tym drugim się nie spotkałem. Linie tego typu można złamać na wiele sposobów. Mój ulubiony to:

```python
if (players[0] == tab[arr[0]] == tab[arr[1]] == tab[arr[2]] or
        players[1] == tab[arr[0]] == tab[arr[1]] == tab[arr[2]]):
```

To jest styl ze standardu kodu do pythona o nazwie `pep8`